src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,19 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        int count = 0;
```

Comment:
let's not forget about checklist before submitting solution
https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html

Reply:
OMG, shame of me)

Reply:
OMG, shame of me)